crates/ingress-rpc/src/service.rs

```diff
@@ -208,8 +210,25 @@ where
             let transaction = self.validate_tx(tx_data).await?;
             total_gas = total_gas.saturating_add(transaction.gas_limit());
         }
-        validate_bundle(&bundle, total_gas, tx_hashes)?;
+        validate_bundle(bundle, total_gas, tx_hashes)?;
 
         Ok(bundle_with_metadata)
     }
+
+    async fn meter_bundle(&self, bundle: &Bundle) -> RpcResult<()> {
+        let res: MeterBundleResponse = self
+            .provider
+            .client()
+            .request("base_meterBundle", (bundle,))
+            .await
+            .map_err(|e| EthApiError::InvalidParams(e.to_string()).into_rpc_err())?;
+
+        // if simulation takes longer than 2s, we don't include and just error to user
+        if res.total_execution_time_us > 2000000 {
```

Comment:
Can we use a constant for this? It'd be even better to pull the value from the superchain registry, but it looks like the Rust codebase [doesn't pull the block time](https://github.com/paradigmxyz/reth/blob/main/crates/optimism/chainspec/src/superchain/chain_metadata.rs#L9).

Reply:
if we're doing the above comment, then this part of code can be removed